### DISCUSSION TITLE
How are we supposed to use `remark-gfm` and `remark-github` together? Footnotes don't work when using both

### DISCUSSION THREAD

**Discussion by swyxio** [remarkjs/remark#1024] [Q&A] [answered]
The docs for [remark-gfm](https://github.com/remarkjs/remark-gfm) and [remark-github](https://github.com/remarkjs/remark-github) mention each other, and of course it should be a very natural thing to want to use both together, but when I do it, it doesnt seem to work: 

![image](https://user-images.githubusercontent.com/6764957/183315533-cd80ab70-f4dc-46f2-8b9a-59402bb38fe9.png)

Produces

![image](https://user-images.githubusercontent.com/6764957/183315542-78c4c379-142c-45dd-a285-262909e6efbb.png)

i know that this works [on github](https://github.com/sw-yx/swyxdotio/issues/440#user-content-fn-1-4be0a98a77381a5806cafa98ee8a628b)

![image](https://user-images.githubusercontent.com/6764957/183315552-6cd4a1b8-4aa4-4ba6-bb4a-7b2263118ab7.png)

so has anyone used both together or what tweaks need to be made? 

i feel like this should be documented at least since i'm pretty sure many people want to use both together

**Comment by ChristianMurphy** [comment]
These plugins are compatible with each other.
My guess is this is related to https://github.com/pngwn/MDsveX/discussions/236
I'd try using earlier versions of these plugins to see if they are compatible with MDSveX.

**Comment by wooorm** [comment, marked as answer]
You also pass them in the same array (with options?), and the other plugin in another array.

That's a bug. remark-gfm does not accept these options.
So move it outside that deep array.

**Reply by wooorm** [reply]
And, a lot of people are using them together, successfully, this is about your particular setup
